File: common/src/main/java/com/andyanika/translator/common/interfaces/usecase/HistoryUseCase.java
package com.andyanika.translator.common.interfaces.usecase;

import com.andyanika.translator.common.models.UiTranslationModel;

import java.util.List;

import io.reactivex.Flowable;

public interface HistoryUseCase {
    Flowable<List<UiTranslationModel>> run(String filter, int limit);
}

File: common/src/main/java/com/andyanika/translator/common/interfaces/usecase/HistoryUseCase.kt
package com.andyanika.translator.common.interfaces.usecase;

import com.andyanika.translator.common.models.FavoriteModel;

import java.util.List;

import io.reactivex.rxjava3.core.Flowable;


public interface HistoryUseCase {
    Flowable<List<FavoriteModel>> run(String filter, int limit);
}
